File: blockchain/structures/transaction.py
from dataclasses import dataclass, field
from typing import Any, Dict

@dataclass
class Transaction:
    v: int
    fee: str
    creator: str
    sig: str
    tx_type: str
    sig_type: str
    nonce: int
    payload: Dict[str, Any] = field(default_factory=dict)

File: blockchain_golang/structures/transaction.go
package structures

type Transaction struct {
	V       uint           `json:"v"`
	Fee     string         `json:"fee"`
	Creator string         `json:"creator"`
	Sig     string         `json:"sig"`
	Type    string         `json:"type"`
	SigType string         `json:"sigType"`
	Nonce   int            `json:"nonce"`
	Payload map[string]any `json:"payload"`
}
